README.md

```diff
@@ -3,10 +3,32 @@
 This project is generated with [yo angular generator](https://github.com/yeoman/generator-angular)
 version 0.15.1.
 
+## Getting Started
+1. git clone
```

Comment:
This should probably be `git clone git@github.com:freecodecamp-fresnoca/arthop-map-dev.git` for ease of use.